test/podcast/create.test.js

```diff
@@ -0,0 +1,33 @@
+const test = require('tape')
+const axios = require('axios')
+const testAPI = require('../testing-api')
+
+test('POST /podcast', async t => {
+  // t.equal(2 + 2, 4, 'addition is working!')
+
+  const api = testAPI()
+
```

Comment:
here we start up our testing api (which runs a temporary database) on port 3001.

we then use a module called `axios` to make a HTTP request to our API